src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs

```diff
@@ -2,40 +2,75 @@
   (:require-macros [status-im.utils.views :as views])
   (:require [re-frame.core :as re-frame]
             [status-im.ui.components.react :as react]
+            [status-im.ui.screens.profile.user.views :as profile]
+            [status-im.utils.build :as build]
+            [status-im.utils.utils :as utils]
+            [status-im.ui.components.colors :as colors]
+            [status-im.i18n :as i18n]
+            [status-im.ui.components.icons.vector-icons :as vector-icons]
+            [taoensso.timbre :as log]
+            [clojure.string :as string]
+            [status-im.ui.components.qr-code-viewer.views :as qr-code-viewer]
+            [status-im.ui.screens.desktop.main.tabs.profile.styles :as styles]
             [status-im.ui.screens.profile.user.views :as profile]))
 
-(defn profile-badge [{:keys [name]}]
-  [react/view {:margin-vertical 10}
-   [react/text {:style {:font-weight :bold}
+(defn profile-badge [{:keys [name photo-path]}]
+  [react/view styles/profile-badge
+   [react/image {:source {:uri photo-path}
+                 :style  styles/profile-photo}]
+   [react/text {:style           styles/profile-user-name
                 :number-of-lines 1}
     name]])
 
-(defn profile-info-item [{:keys [label value]}]
-  [react/view
-   [react/view
-    [react/text
-     label]
-    [react/view {:height 10}]
-    [react/text {:number-of-lines 1
-                 :ellipsizeMode   :middle}
-     value]]])
+(views/defview copied-tooltip [opacity]
+  (views/letsubs []
+    [react/view {:style (styles/tooltip-container opacity)}
+     [react/view {:style styles/tooltip-icon-text}
+      [vector-icons/icon :icons/check
+       {:style styles/check-icon}]
+      [react/text {:style {:font-size 14 :color colors/tooltip-green-text}}
+       (i18n/label :sharing-copied-to-clipboard)]]
+     [react/view {:style styles/tooltip-triangle}]]))
 
-(defn my-profile-info [{:keys [public-key]}]
-  [react/view
-   [profile-info-item
-    {:label "Contact Key"
-     :value public-key}]])
+(views/defview qr-code []
+  (views/letsubs [{:keys [public-key]} [:get-current-account]
+                  tooltip-opacity      [:get-in [:tooltips :qr-copied]]]
+    [react/view
+     #_[react/view {:style styles/close-icon-container}
```

Comment:
`#_` seems like this expression is discarded, why not remove it?

Reply:
Originally comes from my PR, can be removed.